src/main/java/nextstep/auth/AuthorizationExtractor.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.auth;
+
+import java.util.Enumeration;
+import javax.servlet.http.HttpServletRequest;
+import nextstep.support.AuthorizationException;
+
+public class AuthorizationExtractor {
+
+    public static final String AUTHORIZATION = "Authorization";
+
+    public static String BEARER_TYPE = "Bearer";
```

Comment:
`BEARER_TYPE`는 변경되는 일이 없어보여요. 상수로 정의되어 좋지 않을까요 ~ ?

```suggestion
    public static final String BEARER_TYPE = "Bearer";
```